frontend/src/composables/use-collection-meta.ts

```diff
@@ -0,0 +1,44 @@
+import { computed } from "vue"
+
+import type { SupportedMediaType } from "~/constants/media"
+import type { CollectionParams } from "~/types/search"
+import { useI18n } from "~/composables/use-i18n"
+import { useProviderStore } from "~/stores/provider"
+
+import type { ComputedRef } from "vue"
+
+export const useCollectionMeta = ({
+  collectionParams,
+  mediaType,
+  i18n,
+}: {
+  collectionParams: ComputedRef<CollectionParams | null>
+  mediaType: SupportedMediaType
+  i18n: ReturnType<typeof useI18n>
+}) => {
+  const pageTitle = computed(() => {
+    const providerStore = useProviderStore()
+
+    if (collectionParams.value?.collection === "creator") {
+      return `${collectionParams.value.creator} | Openverse`
```

Comment:
For example, in RTL this might also need to be changed? I'm not sure how a browser configured for RTL and with `dir="rtl"` set on a page will interpret a browser title that isn't formatted RTL? Maybe page titles cannot be RTL at all, do you know?